libkbfs/backpressure_disk_limiter.go

```diff
@@ -229,6 +229,89 @@ func (bt *backpressureTracker) getStatus() backpressureTrackerStatus {
 	}
 }
 
+// quotaBackpressureTracker keeps track of the variables used to
```

Comment:
I think the existing backpressureTracker shouldn't be used for quota backpressure because:

1) We don't actually want to set a hard limit, i.e. use a semaphore, for the same reason the quota isn't a hard quota, in that we still want writes to go through so that the user can delete files.
2) We slow down writes after the quota is reached, right? It seems we want to simulate that locally, so the delay calculation is different.

What do you think?